source/_components/light.hive.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: "Hive Light"
+description: "Instructions on how to integrate Hive lights with Home Assistant."
+date: 2017-09-24 21:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hive.png
+ha_category: Light
+ha_release: 0.62
+ha_iot_class: "Cloud Polling"
+---
+
+
+The 'hive' light component integrates your Hive lights in to Home Assistant, enabling control of various settings, depending on model light.
```

Comment:
Typo: `in to` -> `into`
Missing determiner: `on the model light`.

Reply:
Updated